src/main.c

```diff
@@ -1152,11 +1153,15 @@ main(int argc, char *argv[])
     np2srv.server_dir = SERVER_DIR;
 
     /* process command line options */
-    while ((c = getopt(argc, argv, "dhVp:f:U::m:u:g:n:i:t:x:v:c:")) != -1) {
+    while ((c = getopt(argc, argv, "dFhVp:f:U::m:u:g:n:i:t:x:v:c:")) != -1) {
         switch (c) {
         case 'd':
             daemonize = 0;
             break;
+	case 'F':
```

Comment:
Is this a TAB? If so, it is the reason why the tests failed.

Reply:
Indeed, it was a TAB, really thought I had cleaned that up. Sorry, fixing!